test/python_client_test.py

```diff
@@ -150,25 +170,44 @@ def test_error(self):
       self.conn.get_srv_keyspace('error')
 
   def test_effective_caller_id(self):
```

Comment:
I think this test should cover all possible cursor APIs in vtgate_cursor.py so that we will be confident that we don't miss any piece of callerid on the server side